src/sdl2_compat.c

```diff
@@ -24,6 +24,7 @@
 #include "sdl3_include_wrapper.h"
 #include "sdl2_compat.h"
 #include "dynapi/SDL_dynapi.h"
+#include "SDL2/SDL_revision.h"
```

Comment:
I'm unsure whether this is desired.

Reply:
It might, but ugh.

Can we not add it to CFLAGS as a new SDL2COMPAT_REVISION define (private to library only) and return that instead of SDL_REVISION?

Reply:
Force pushed a fix.
When no `SDL2COMPAT_REVISION` is passed, it will return `"SDL-2.90.0-unknown"`.

Reply:
Looks good to me.

If have to be picky: Maybe `-no-vcs` instead of `-unknown` to follow our showrev.sh?? I'm fine either way, though.